src/data_model/math2eng.jl

```diff
@@ -0,0 +1,209 @@
+""
+function transform_solution(solution_math::Dict{String,<:Any}, data_math::Dict{String,<:Any}; map::Union{Vector{Dict{String,<:Any}},Missing}=missing, make_si::Bool=true)::Dict{String,Any}
```

Comment:
I would add `convert_rad2deg=true` as a keyword argument here and pass it to `solution_make_si`